test_suites/vp9/VP9-TEST-VECTORS-HIGH.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "VP9-TEST-VECTORS-HIGH",
+    "codec": "VP9",
+    "description": "VP9 High Bit Depth Test Vectors from https://storage.googleapis.com/downloads.webmproject.org/test_data/libvpx/",
+    "test_vectors": [
+        {
```

Comment:
We have three tests left behind:

```
013708bd043f0821a3e56fb8404d82e7a0c7af6c *vp91-2-04-yuv422.webm
1e58a7d23adad830a672f1733c9d2ae17890d59c *vp91-2-04-yuv422.webm.md5
25d78f28948789d159a9453ebc13048b818251b1 *vp91-2-04-yuv440.webm
81b3870b27a7f695ef6a43e87ab04bbdb5aee2f5 *vp91-2-04-yuv440.webm.md5
0321d507ce62dedc8a51b4e9011f7a19aed9c3dc *vp91-2-04-yuv444.webm
367e423dd41fdb49aa028574a2cfec5c2f325c5c *vp91-2-04-yuv444.webm.md5
```
Basicaly, you added tests for profile 2 and 3, but skipped profile 1. 2 and 3 is high bitdepth, so when you say high in the name of the suite, that's what you mean ?

Reply:
I missed those. By high bit depth I mean the ones behind CONFIG_VP9_HIGHBITDEPTH in test/test-data.mk

Reply:
Also no 440 videos seem to work, i.e.: vp91-2-04-yuv440.webm, vp93-2-20-10bit-yuv440.webm, vp93-2-20-12bit-yuv440.webm do not work.

I.e.: Firefox/chromium/gstreamer won't play those. ffmpeg outputs some crazy stuff too

Reply:
I know what 4:0:0 is, but 4:4:0 ?

Reply:
Explained here, https://blog.csdn.net/qq_39565868/article/details/114877172